pycodestyle.py

```diff
@@ -120,7 +120,7 @@
 LAMBDA_REGEX = re.compile(r'\blambda\b')
 HUNK_REGEX = re.compile(r'^@@ -\d+(?:,\d+)? \+(\d+)(?:,(\d+))? @@.*$')
 
-# Work around Python < 2.6 behaviour, which does not generate NL after
+# Work around Python < 2.6 behavior, which does not generate NL after
```

Comment:
This is actually a difference between American English and British English. I don't consider it a typo.